antlr-kotlin-gradle-plugin/src/main/java/com/strumenta/antlrkotlin/gradleplugin/AntlrKotlinPlugin.java

```diff
@@ -74,9 +74,11 @@ private void setupClasspath(final Project project) {
             defaultDependencies.add(projectDependencies
                     .create("com.strumenta.antlr-kotlin:antlr-kotlin-target:" + antlrKotlinVersion));
 
-            LOGGER.info("using antlr-kotlin-runtime-{} version {}", platform, antlrKotlinVersion);
-            defaultDependencies.add(projectDependencies
-                    .create("com.strumenta.antlr-kotlin:antlr-kotlin-runtime-" + platform + ":" + antlrKotlinVersion));
+            // FIXME: removed it until I figure out how to make it work for Multiplatform project.
```

Comment:
At the very least we should remember to open an issue for this. To me it seems an issue not having the dependency